README.md

```diff
@@ -0,0 +1,61 @@
+[![Build Status](https://travis-ci.org/xquery/curl-docker.svg?branch=master)](https://travis-ci.org/curl/curl-docker)
+
+# Curl Docker Images repository
+
+This repository defines internal & official curl docker images.
+
+* [alpine](alpine/)
```

Comment:
Why centos and debian? Why not just alpine? Why not just scratch? An `alpine` docker works fine on Linux; you don't need to match to the host OS.

Reply:
I wanted to ensure repo could handle multiples and arbitrarily picked these three. I do think there are some people who prefer their docker containers to have a specific dist ... minimally for comprehending/setting runtime configuration ... I am equally fine with settling on a single dist though I still think we would need a windows version.

Reply:
Docker on Windows uses virtualization under the covers, so it can use Linux images.

You know my preference, which is to try and make this a scratch container and link to musl :smile: but settling on just alpine is a good enough start to be honest.